### DISCUSSION TITLE
%EB%B3%BC%ED%83%80-%EC%9D%B4%EC%95%BC%EA%B8%B0-%EC%B1%84%EC%9A%A9-%EC%9B%90%EC%B9%99/

### DISCUSSION THREAD

**Discussion by giscus** [heli-os/profile-giscus#7] [Comments]
# %EB%B3%BC%ED%83%80-%EC%9D%B4%EC%95%BC%EA%B8%B0-%EC%B1%84%EC%9A%A9-%EC%9B%90%EC%B9%99/

볼타 CTO 테오 블로그

https://dataportal.kr/%EB%B3%BC%ED%83%80-%EC%9D%B4%EC%95%BC%EA%B8%B0-%EC%B1%84%EC%9A%A9-%EC%9B%90%EC%B9%99/

**Comment by gosekose** [comment]
훌륭한 팀원분들과 함께 프로덕트를 만들어 회사와 함께 성장하는게 정말 큰 기쁨인거 같습니다!! 잘보고 갑니다~~
